### ISSUE TITLE
NullPointerException if the meta directory does not exist

### ISSUE THREAD

**Issue by Sorixelle** [gwen-interpreter/gwen#61]
If the user has deleted the `gwen/meta/` directory in a project created by `gwen init`, Gwen instantly crashes on startup with the following exception:
```
ERROR - NullPointerException
java.lang.NullPointerException
        at scala.collection.ArrayOps$.filter$extension(ArrayOps.scala:191)
        at gwen.core.FileIO$.recursiveScan(Predefs.scala:167)
        at gwen.core.eval.GwenLauncher.$anonfun$1(GwenLauncher.scala:108)
        at scala.collection.immutable.List.flatMap(List.scala:293)
        at gwen.core.eval.GwenLauncher.run(GwenLauncher.scala:108)
        at gwen.GwenInterpreter.run(GwenInterpreter.scala:88)
        at gwen.GwenInterpreter.main(GwenInterpreter.scala:64)
        at gwen.web.GwenWebInterpreter.main(GwenWebInterpreter.scala)
```

**Comment by bjuric** [comment]
Fixed in gwen-web v[3.0.7](https://github.com/gwen-interpreter/gwen-web/releases/tag/v3.0.7) Close when ready. Thanks @Sorixelle.

**Comment by Sorixelle** [comment]
Looks to be resolved.
